module/input/GameController/src/GameController.cpp

```diff
@@ -86,7 +86,7 @@ namespace module::input {
 
                           // Bind our new handle
                           std::tie(listenHandle, std::ignore, std::ignore) =
-                              on<UDP::Broadcast, With<GameState>>(recieve_port)
+                              on<UDP::Broadcast, With<GameState>, Single>(recieve_port)
```

Comment:
There'll be other instances :p 
```suggestion
                              on<UDP::Broadcast, With<GameState>, Single>(receive_port)
```